tests/framework/validators/FileValidatorTest.php

```
@@ -816,9 +818,85 @@ public function testValidateTypedAttributeMaxError(): void
+        $file = new UploadedFile(['error' => 999, 'name' => 'test.txt']);
+        $result = $this->invokeMethod($val, 'validateValue', [$file]);
+        $this->assertEquals([$val->message, []], $result);
+    }
+
+    public function testValidateExtensionEdgeCases(): void
+    {
+        $val = new FileValidator(['extensions' => ['jpg']]);
+
+        $file = new UploadedFile(['tempName' => '/non/existent', 'name' => 'test.jpg']);
+        $this->assertFalse(@$this->invokeMethod($val, 'validateExtension', [$file]));
+    }
```

PHPMD (2.15.0)

[error] Remove error control operator '@' on line 859. (undefined)

(ErrorControlOperator)